extensions/SemanticForms/specials/SF_FormStart.php

```diff
@@ -133,7 +133,7 @@ function doRedirect( $form_name, $page_name, $params ) {
 		if ( $page_title->exists() ) {
 			// It exists - see if page is a redirect; if
 			// it is, edit the target page instead.
-			$content = WikiPage::factory( $page_title )->getContent();
+			$content = WikiPage::factory( $page_title ); # Wikia change - SUS-1271 / use MW1.19 compatible call
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)